browser/resources/resource_ids

```diff
@@ -88,4 +85,11 @@
   "<(ROOT_GEN_DIR)/brave/web-ui-brave_wallet/brave_wallet.grd": {
     "includes": [43600],
   },
+  "brave/components/brave_extension/extension/resources.grd": {
```

Comment:
these were moved because they were out of order and didn't have a large enough range